src/bh_route_stats.erl

```diff
@@ -26,7 +26,7 @@ prepare_conn(Conn) ->
     bh_db_worker:load_from_eql(Conn, "stats.sql", Loads).
 
 handle('GET', [], _Req) ->
-    ?MK_RESPONSE(get_stats(), {block_time, 5});
+    ?MK_RESPONSE(get_stats(), {block_time, 1});
```

Comment:
The main reason to not have such short cache times is that this change 5x's the number of requests and corresponding not super cheap sql queries.